frontend/src/components/ClickableTooltip.vue

```diff
@@ -0,0 +1,42 @@
+<template>
+  <q-btn-dropdown
```

Comment:
Can we make the tooltip also show up on hover? I feel like hover for desktop and tap for mobile is the standard approach.

Also, if we could replace other uses of `q-tooltip` with this custom component in this PR, that way everything is consistent, that would be awesome